trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/get-and-clear-redirect-session-data.ts

```diff
@@ -0,0 +1,39 @@
+import { Request } from 'express';
+import { ErrorSummaryViewModel } from '../../../types/view-models';
+import { getAddPaymentError } from '../helpers/get-add-payment-error-helper';
+
+const clearRedirectSessionData = (req: Request): void => {
+  delete req.session.addPaymentErrorKey;
+  delete req.session.checkedCheckboxIds;
+};
+
+export const getAndClearRedirectSessionData = (
```

Comment:
I feel this would be best named in a way that is related to the page, I know it lives in the folder but it otherwise reads very generically but isn't which could be a source of confusion